Dockerfile

```diff
@@ -1,16 +1,15 @@
+ARG OS=linux
+ARG ARCH=amd64
 FROM quay.io/bitnami/golang:1.17 AS builder
 WORKDIR /go/src/open-cluster-management.io/registration
 COPY . .
 ENV GO_PACKAGE open-cluster-management.io/registration
 
-RUN make build --warn-undefined-variables
-RUN make build-e2e --warn-undefined-variables
+RUN GOOS=${OS} GOARCH=${ARCH} make build --warn-undefined-variables
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
replace this w/ a multi-arch base image?

Reply:
I have to pull base image at first, since there is no separate tag for different platform